<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstForm</class>
 <widget class="QWidget" name="FirstForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>184</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>279</width>
    <height>184</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>279</width>
    <height>184</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Заполнение анкеты</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Данные об изделии</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>261</width>
     <height>138</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Тип изделия</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Металлическое изделие</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Деревянное изделие</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Другое</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Название изделия</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="nameprod_edit"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Материал</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="3">
     <widget class="QLineEdit" name="material_edit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Вес</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QLineEdit" name="weight_edit"/>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Починка</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>Далее</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
